app/models/turbo/streams/tag_builder.rb

```diff
@@ -77,8 +77,8 @@ def remove_all(targets)
   #   <%= turbo_stream.replace "clearance_5" do %>
   #     <div id='clearance_5'>Replace the dom target identified by clearance_5</div>
   #   <% end %>
-  def replace(target, content = nil, **rendering, &block)
-    action :replace, target, content, **rendering, &block
+  def replace(target, content = nil, method: nil, **rendering, &block)
```

Comment:
It's not currently clear from the docs here what the `method` parameter is for. One needs to go to https://turbo.hotwired.dev/reference/streams#replace to understand. Maybe it would be useful to have a example for it in the method docs?

```suggestion
#   <%= turbo_stream.replace clearance, "<div>Morph the dom target</div>", method: :morph %>
  def replace(target, content = nil, method: nil, **rendering, &block)
```

Reply:
DONE